docs/modules/operation/pages/service-assurance/monitors/DiskUsageMonitor.adoc

```diff
@@ -1,51 +1,55 @@
 
 = DiskUsageMonitor
 
-The DiskUsageMonitor monitor can be used to test the amount of free space available on certain storages of a node.
-The monitor gets information about the available free storage spaces available by inspecting the _hrStorageTable_ of the link:http://tools.ietf.org/html/rfc1514[HOST-RESOURCES-MIB].
-A storage's description (as found in the corresponding _hrStorageDescr_ object) must match the criteria specified by the `disk` and `match-type` parameters to be monitored.
-A storage's available free space is calculated using the corresponding _hrStorageSize_ and _hrStorageUsed_ objects.
+The DiskUsageMonitor tests the amount of free space available on certain node storage.
+The monitor inspects the http://tools.ietf.org/html/rfc1514[HOST-RESOURCES-MIB] hrStorageTable to get information about the available free storage space.
 
-CAUTION: The _hrStorageUsed_ doesn't account for filesystem reserved blocks (i.e., for the super-user), so DiskUsageMonitor will report the service as 
+A storage's description (as found in the corresponding hrStorageDescr object) must match the criteria the `disk` and `match-type` parameters specify to be monitored.
+A storage's available free space is calculated using the corresponding hrStorageSize and hrStorageUsed objects.
+
+CAUTION: The hrStorageUsed does not account for filesystem reserved blocks (for example, for the super-user), so DiskUsageMonitor will report the service as 
          unavailable only when the amount of free disk space is actually lower than `free` minus the percentage of reserved filesystem blocks.
 
-This monitor uses _SNMP_ to accomplish its work.
-Therefore systems against which it is to be used must have an SNMP agent supporting the _HOST-RESOURCES-MIB_ installed and configured.
-Most modern _SNMP agents_, including most distributions of the _Net-SNMP agent_ and the _SNMP service_ that ships with _Microsoft Windows_, support this _MIB_.
-Out-of-box support for _HOST-RESOURCES-MIB_ among commercial _Unix_ operating systems may be somewhat spotty.
+This monitor uses SNMP to accomplish its work.
+Therefore, systems against which you use it must have an SNMP agent supporting the HOST-RESOURCES-MIB installed and configured.
+Most modern SNMP agents, including most distributions of the _Net-SNMP agent_ and the _SNMP service_ that ships with Microsoft Windows, support this MIB.
+Out-of-the-box support for HOST-RESOURCES-MIB among commercial Unix operating systems may be spotty.
 
-== Monitor Facts
+== Monitor facts
 
 [options="autowidth"]
 |===
 | Class Name     | `org.opennms.netmgt.poller.monitors.DiskUsageMonitor`
-| Remote Enabled | false, relies on SNMP configuration.
+| Remote Enabled | false, relies on SNMP configuration
```

Comment:
Is this true?  Shouldn't this work remotely via Minions?

Reply:
@indigo423, do you know the answer to this?

Reply:
Chandra confirmed it should be "true".